frontend/benefit/applicant/src/components/applications/applicationList/listItem/ListItem.tsx

```diff
@@ -47,57 +53,87 @@ const ListItem: React.FC<ListItemProps> = (props) => {
-          </$DataColumn>
-        )}
-        {statusText && (
+    <$ListItemWrapper>
+      <$ListItem>
+        <$ItemContent>
+          <$Avatar $backgroundColor={avatar.color}>{avatar.initials}</$Avatar>
           <$DataColumn>
-            <$DataHeader>{t(`${translationListBase}.status`)}</$DataHeader>
-            <$DataValue>{statusText}</$DataValue>
+            <$DataHeader>{t(`${translationBase}.common.employee`)}</$DataHeader>
+            <$DataValue>{name}</$DataValue>
           </$DataColumn>
-        )}
-      </$ItemContent>
-      <$ItemActions>
-        <Button
-          variant="secondary"
-          iconLeft={ActionIcon && <ActionIcon />}
-          theme="black"
-          onClick={allowedAction.handleAction}
-          fullWidth
-        >
-          {allowedAction.label}
-        </Button>
-      </$ItemActions>
-    </$ListItem>
+          {createdAt && (
+            <$DataColumn>
+              <$DataHeader>
+                {t(`${translationBase}.common.created`)}
+              </$DataHeader>
+              <$DataValue>{createdAt}</$DataValue>
+            </$DataColumn>
+          )}
+          {submittedAt && (
+            <$DataColumn>
+              <$DataHeader>{t(`${translationBase}.common.sent`)}</$DataHeader>
+              <$DataValue>{submittedAt}</$DataValue>
+            </$DataColumn>
+          )}
+          {applicationNum && (
+            <$DataColumn>
+              <$DataHeader>
+                {t(`${translationBase}.common.applicationNumber`)}
+              </$DataHeader>
+              <$DataValue>{applicationNum}</$DataValue>
+            </$DataColumn>
+          )}
+          {statusText && (
+            <$DataColumn>
+              <$DataHeader>{t(`${translationBase}.common.status`)}</$DataHeader>
+              <$DataValue>{statusText}</$DataValue>
+            </$DataColumn>
+          )}
+          {editEndDate && (
+            <$DataColumn>
+              <$DataHeader>
+                {t(`${translationBase}.common.editEndDate`)}
+              </$DataHeader>
+              <$DataValueAlert>{editEndDate}</$DataValueAlert>
+            </$DataColumn>
+          )}
+        </$ItemContent>
+        <$ItemActions>
+          <Button
+            variant={
+              status === APPLICATION_STATUSES.INFO_REQUIRED
+                ? 'primary'
+                : 'secondary'
+            }
+            iconLeft={ActionIcon && <ActionIcon />}
+            theme={
+              status === APPLICATION_STATUSES.INFO_REQUIRED ? 'coat' : 'black'
+            }
+            onClick={allowedAction.handleAction}
+            fullWidth
+          >
+            {allowedAction.label}
+          </Button>
+        </$ItemActions>
+      </$ListItem>
+      {status === APPLICATION_STATUSES.INFO_REQUIRED && (
+        <$ItemInfo>
+          <$GridCell $colStart={2}>
+            <$ItemInfoInner>
+              <IconSpeechbubbleText />
+              <$ItemInfoText>1 uusi viesti</$ItemInfoText>
```

Comment:
Why don't we add `uusi viesti` with its plural form already to the translation files?

Reply:
we can yes